js/client/modules/@arangodb/testutils/test-utils.js

```diff
@@ -632,11 +200,11 @@ function splitBuckets (options, cases) {
     return cases;
   }
   if (cases.length === 0) {
-    didSplitBuckets = true;
+    didSplitBuckets(true);
     return cases;
   }
 
-  didSplitBuckets = true;
+  didSplitBuckets(didSplitBuckets);
```

Comment:
This line of code puzzles me a bit. What kind of magic is going on here?

Reply:
its moving the location of this static variable.